tests/error_message.rs

```diff
@@ -0,0 +1,17 @@
+extern crate snafu;
+
+use std::error::Error as _;
```

Comment:
I'm betting the `as _` won't work in the older Rust versions, but I don't think you need it either way.